backend/src/apps/owasp/migrations/0076_remove_certificate_unique_active_cert_per_tier_and_more.py

```
@@ -0,0 +1,63 @@
+# Generated by Django 6.0.8 on 2026-08-13 09:03
+
+import django.db.models.deletion
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('github', '0044_user_indexes'),
+        ('owasp', '0075_alter_certificate_id'),
+    ]
```

Ruff (0.16.1)

[warning] Mutable default value for class attribute

(RUF012)